src/app/js/data/blocks-split.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>TileJSON</title>
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!-- Compiled and minified CSS -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
+    <link rel="stylesheet" href="https://openlayers.org/en/v4.3.3/css/ol.css" type="text/css">
+    <!-- The line below is only needed for old environments like Internet Explorer and Android 4.x -->
+    <script src="https://cdn.polyfill.io/v2/polyfill.min.js?features=requestAnimationFrame,Element.prototype.classList,URL"></script>
+    <script src="https://openlayers.org/en/v4.3.3/build/ol.js"></script>
+    <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"></script>
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+<body>
+    <style>
+        html, body, #container {
+            margin: 0;
+            padding: 0;
+            height: 100%;
+            width: 100%;
+            max-height: 100%;
+            max-width: 100%;
+        }
+
+        #map {
+            position: absolute;
+            top: 116px;
+            left: 0px;
+            width: 100%;
+            height: calc(100% - 116px);
+            z-index: 500;
+        }
+
+        #overDiffMap {
+            position: absolute;
+            top: 132px;
+            left: 0px;
+            width: 100%;
+            max-height: calc(100% - 116px);
+            z-index: 1000;
+        }
+
+        #card {
+            position: absolute;
+            top: 180px;
+            right: 16px;
+            width: 320px;
+            z-index: 2000;
+        }
+
+        #swipe {
+            -webkit-appearance: none;
+            appearance: none;
+            width: 100%;
+            height: 0px;
+            opacity: 0.7;
+            border: none;
+        }
+
+        #swipe:hover {
+            opacity: 1;
+        }
+
+        #swipe::-webkit-slider-runnable-track {
+            height: 28px;
+            background-color: transparent;
+            border: none;
+        }
+
+        #swipe::-webkit-slider-thumb {
+            background: #ee6e73;
+            height: 32px;
+            width: 32px;
+        }
+
+        #layer-toolbar {
+            background-color: #e8e8e8;
+            height: 52px;
+            margin-bottom: 0px;
+        }
```

Comment:
Not sure if this is possible or makes sense, but if this CSS is identical to CSS used on the site, can it be stored in a separate file that can be referenced by the site and POSTed/used within the gist? That way we wouldn't have duplicate code.